docs/api/swagger.json

```diff
@@ -69,47 +75,927 @@
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "name": {
+            "type": "string"
+          },
+          "description": {
+            "type": "string"
+          },
+          "active": {
+            "type": "boolean"
+          },
+          "translations": {
+            "type": "array",
+            "items": {
+              "properties": {
+                "name": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      },
+      "VocabularyQueried": {
+        "description": "Vocabulary",
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "stage": {
+            "type": "integer"
+          },
+          "name": {
+            "type": "string"
           },
-          "400": {
-            "$ref": "#/components/responses/400"
+          "description": {
+            "type": "string"
           },
-          "401": {
-            "$ref": "#/components/responses/401"
+          "translations": {
+            "type": "array",
+            "items": {
+              "properties": {
+                "name": {
+                  "type": "string"
+                }
+              }
+            }
           }
         }
-      }
-    }
-  },
-  "components": {
-    "schemas": {
-      "User": {
-        "description": "User",
+      },
+      "Progress": {
+        "description": "Progress",
         "type": "object",
         "properties": {
-          "id": {
-            "type": "string",
-            "format": "uuid"
+          "queryProgress": {
+            "type": "object",
+            "properties": {
+              "dueToday": {
+                "type": "string"
+              },
+              "correct": {
+                "type": "string"
+              },
+              "wrong": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      },
+      "Info": {
+        "description": "Info",
+        "type": "object",
+        "properties": {
+          "identifier": {
+            "type": "string"
           },
-          "username": {
-            "type": "string",
-            "nullable": true
+          "version": {
+            "type": "string"
           },
-          "email": {
-            "type": "string",
-            "format": "email"
+          "commitRef": {
+            "type": "string"
+          }
+        }
+      },
+      "Language": {
+        "description": "Language",
+        "type": "object",
+        "properties": {
+          "code": {
+            "type": "string"
+          },
+          "name": {
+            "type": "string"
+          },
+          "nativeNames": {
+            "type": "array",
+            "items": {
+              "name": {
+                "type": "string"
+              }
+            }
```

Comment:
array cannot have name property with object. Change to following to fix this error:
![image](https://user-images.githubusercontent.com/60048565/129562760-21a6fa5a-1844-4b30-a676-e6415a7e2c9f.png)

```suggestion
            "items": {
              "type": "string"
            }
```